bucket/eac.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.3",
+    "description": "A free audio CD grabber using standard drives.",
+    "homepage": "http://www.exactaudiocopy.de/",
+    "license": "Freeware",
+    "url": "http://www.exactaudiocopy.de/eac-1.3.exe#dl.7z",
+    "hash": "sha1:49f1028bd2b0cce0829250430bf8771c4a766daf",
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Empty*\", \"$dir\\uninst*\" -Recurse",
+    "shortcuts": [
+        [
+            "EAC.exe",
+            "Exact Audio Copy"
+        ]
+    ],
+    "checkver": {
```

Comment:
Is it needed? 

> Exact Audio Copy V1.3 (including CDRDAO and Flac packages) last changed on **04.09.16**

Reply:
I suggest it may update in the future, as FreeDB, the CD metadata library it depends, is shutting down.